pharmd-app/src/screens/upload/components/SpreadsheetUploader.js

```diff
@@ -105,13 +82,6 @@ class SpreadsheetUploader extends Component {
                 {!this.state.uploadedData &&
```

Comment:
I can't comment directly ;-; but if you expand the untouched code above in this file there's a todo saying to replace with modal dialog

Reply:
`  // TODO: replace with modal dialog`